src/components/dropdown-item/dropdown-item.tsx

```diff
@@ -13,9 +13,45 @@ export class DropdownItem {
   @Prop() disabled: boolean;
   @Prop() icon: string;
   @Prop() checkable: boolean = false;
+  @Prop() value: string = ""
   @State() checkboxColor: string = "";
+  @Event({ bubbles: false }) itemValues: EventEmitter<Object>;
   @Element() el;
 
+  // handleMenuItem() {
+  //   const target = this.el.innerHTML;
+  //   //console.log('target', this.el.innerHTML)
+  //   this.select.emit(target);
+  // }
+
+  // handleCheckValue(event) { 
+  //   this.ItemCheck.emit(event.target.checked)
+  // }
+
+  toggleCheckBox() { 
+    let input = this.el.shadowRoot.querySelector('input')
+    if(input) { 
+      input.checked = !input.checked
+    }
+  }
+
+  handleItemChange(event) { 
+    const isNested = this.el.closest('ifx-dropdown')
+    if(!isNested) { 
+      this.toggleCheckBox()
+    }
+    //console.log('current Target',event.currentTarget)
+    if(event.currentTarget.className.toLowerCase() !== 'form-check-input') { 
+      
+      const checkBoxValue = this.el.shadowRoot.querySelector('input');
+      if(checkBoxValue) { 
+        this.itemValues.emit({check: checkBoxValue.checked, value: this.value})
```

Comment:
where is this.value being set?

Reply:
from the value attribute of the <option> element. 
<img width="251" alt="value" src="https://github.com/Infineon/infineon-design-system-stencil/assets/108734092/bcd68b44-4819-4b65-9ab5-e46a6ecafac5">